js/apps/system/_admin/aardvark/APP/react/src/utils/useDisableNavBar.ts

```diff
@@ -8,14 +8,14 @@ import { useEffect } from "react";
 export const useDisableNavBar = () => {
   useEffect(() => {
     const bottomSubNameSelector = "#subNavigationBar .subMenuEntries.bottom";
-    const bottomSubNav = $(bottomSubNameSelector);
+    const bottomSubNav = window.$(bottomSubNameSelector);
```

Comment:
@pluma4345 we started getting some TS errors because `$` was not recognized, so I added this (as we have a `any` type for `window`). Not sure why it came up now. We can move away from jquery but probably no tin this PR as it's already doing a lot. @pluma4345